server/server.go

```diff
@@ -584,6 +584,11 @@ func (s *Server) startReloadWatcher(ctx context.Context) {
 			// pinned and sandbox/pause images, we need to update them
 			s.StorageImageServer().UpdatePinnedImagesList(append(s.config.PinnedImages, s.config.PauseImage))
 			logrus.Info("Configuration reload completed")
+			// Print the current config
+			if err := s.config.LogConfig(); err != nil {
+				logrus.Debugf("Unable to log configuration: %v", err)
+				continue
+			}
```

Comment:
Do we need to print the configuration here? Not sure if it's worth doing it here.

Reply:
I think this was to have the config printed at the event of reload.

Reply:
Which one?

- https://github.com/cri-o/cri-o/issues/7586
- https://github.com/cri-o/cri-o/issues/7704

I think neither has such a requirement. Unless I missed something there in the comments and such?

Reply:
Maybe it was this comment that made me think so https://github.com/cri-o/cri-o/issues/7704#issuecomment-1907720363
We were also discussing it @sohankunkerkar